modules/exploits/windows/misc/allmedia_1_6_buffer_overflow.rb

```diff
@@ -0,0 +1,89 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+# Author: Hejap Zairy
+# Date: 1.08.2022
+# Exploit Prof
+# Proof and Exploit:
+#image:https://i.imgur.com/yLrRR2t.png
+#video:https://streamable.com/x4i50c
+
+
+
+require 'msf/core'
+
+class Metasploit4 < Msf::Exploit::Remote
+    Rank = ExcellentRanking
+
+    include Msf::Exploit::Remote::Tcp
+    include Msf::Exploit::Seh
+
+    def initialize(info = {})
+        super(update_info(info,
+            'Name'           => 'ALLMediaServer 1.6  Buffer Overflow',
+            'Description'    => %q{
+                This module exploits a stack buffer overflow in ALLMediaServer 1.6
+                The vulnerability is caused due to a boundary error within the
+                handling of HTTP request.
+                Thank you Saud Alenazi and 0xSaudi
+                and Muhammad Al Ahmadi and all the friends in Tuwaiq i Love Tuwaiq
```

Comment:
```suggestion
```

Whilst I appreciate you shouting out friends, this is not the place to do it. Feel free to do that in the PR description or as a comment on this PR, however the description here should only contain details about the bug itself and expected behavior of the module.